src/content/docs/style-guide/images/embed-images.mdx

```diff
@@ -15,61 +15,114 @@ redirects:
   - /docs/nronly/images
   - /docs/style-guide/processes-procedures/edit-images
   - /docs/style-guide/processes-procedures/embed-images
+resources:
+  - title: Image codemod README
+    url: 'https://github.com/newrelic/docs-website/blob/b43d853f9ba4b17d4c090fb0a650b8f18cb0841d/codemods/convertMDXImages/README.md'
 ---
 
 A well-chosen screenshot or image can greatly improve the readability and clarity of a doc. Too many images or an image that's tough to parse can really slow things down.
 
-* If you're not part of the docs team and you want to add an image to the docs site, create a [GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose).
-* If you're a New Relic employee, contact the `@hero` in the documentation Slack channel.
+- If you're not part of the docs team and you want to add an image to the docs site, create a [GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose).
+- If you're a New Relic employee, contact the `@hero` in the documentation Slack channel.
 
 ## Add an image [#add]
 
 We want to ensure that sensitive data does not appear in UI screenshots. That's why we recommend that you provide a permalink to a test account. Do not take a screenshot from your New Relic account.
 
 Here are some things to keep in mind when you're creating an image:
 
-* Make sure your image provides useful information at a glance.
-* Include a caption with helpful context for the image.
-* For screenshot captions, include the path in bold.
-* For video captions, include the approximate running time.
+- Make sure your image provides useful information at a glance.
+- Use alphanumeric characters in combo with `-` or `_` in the image file name. Avoid using any other special characters.
+- Include a caption with helpful context for the image.
+- For screenshot captions, include the path in bold.
+- For video captions, include the approximate running time.
 
-Our doc site images are stored in individual `images` directories at the root level of each taxonomy category. These `images` directories contain all the images used in the docs for that category. Do not link to an external file, all images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
+Do not link to an external file, all images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
```

Comment:
I don't think a comma here is grammatically correct. Recommend change to a period "...file. All images..." or else a semi-colon.